### DISCUSSION TITLE
Can DsHidMini accurately determine whether a PS3 controller is fake or genuine ?

### DISCUSSION THREAD

**Discussion by l3gi0n0fh3ll** [nefarius/DsHidMini#137] [Q&A] [answered]
Can DsHidMini mistake a fake PS3 controller for a genuine DS3 one ? Is it accurate ?

Also, why are the controller led patterns changed when I install DsHidMini ?

What is the led pattern when the controller is being charged and when it is fully charged ?

**Comment by nefarius** [comment, marked as answer]
> Can DsHidMini mistake a fake PS3 controller for a genuine DS3 one ? Is it accurate ?

Explained in great detail here #3 

> Also, why are the controller led patterns changed when I install DsHidMini ?

Explained in README.

> What is the led pattern when the controller is being charged and when it is fully charged ?

Explained in README and the [docs site](https://vigem.org/projects/DsHidMini/HID-Device-Modes-Explained/).

Cheers
